app/controllers/api/v1/occupation_standard_skills_controller.rb

```diff
@@ -9,7 +11,31 @@ def index
   end
 
   def show
+    render_resource
+  end
+
+  def update
+    authorize [:api, :v1, @oss]
+    skill = Skill.where(update_params).first_or_initialize(parent_skill: @oss.skill)
+    if skill.save
+      @oss.update(skill: skill)
+      render_resource
+    else
+      render_resource_error(skill)
+    end
+  end
+
+  private
+
+  def set_occupation_standard_skill
     @oss = OccupationStandardSkill.find(params[:id])
+  end
+
+  def update_params
+    params.require(:data).require(:attributes).permit(:description)
+  end
+
+  def render_resource
```

Comment:
This doesnt need to block this PR, but it seems like we should a.) make this `render_object` so its consistent with `occupation_standards_controller.rb` and potentially b.) abstract it out to something that's shared in `API::V1` controllers.

Reply:
Dont feel like you need to take that on b. unless it feels clearly better... Typing it out below to see for myself how I feel about it... 

`render_resource(@oss, `OccupationStandardSkill`, resource_options)`

with

`
def resource_options
  { links: ... }
end
`

and in `API::V1::APIController`:

`
def render_object(object, serializer, options)
  render json: "API::V1::#{serializer}Serializer".constantize.new(object, options)
end
`

Maybe? Feel free top punt on this until we have more examples in our controllers.

Reply:
Good point about consistent naming, and refactoring! I can make those changes here.

Reply:
@Pcushing actually, I'm just going to rename the method for consistency for now. The refactoring is more complicated in the `OccupationStandardsController` since both the index method and the show method call `render_resource` but have different options.  But I agree it should get refactored down the line.